src/sanity.ts

```diff
@@ -0,0 +1,779 @@
+const completionSpec: Fig.Spec = {
+  name: "sanity",
+  description: "",
```

Comment:
Not sure how I missed this the first time 🤦 should be a quick fix, can just edit the PR directly if you like.
```suggestion
  description: "Sanity CLI tool for managing Sanity installations, managing plugins, schemas and datasets",
```

Reply:
Actually I'll just add this right now so we can merge 😃